src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -12,9 +16,48 @@ const DefiHeader = () => {
   )
 }
 
+const FoxFarmCTA = () => {
+  const ethLogoSrc = 'https://assets.coincap.io/assets/icons/256/eth.png'
+  const foxLogoSrc = 'https://assets.coincap.io/assets/icons/256/fox.png'
+  return (
+    <Card variant='outline' my={1}>
+      <Stack
+        direction={{ base: 'column', xl: 'row' }}
+        alignItems='center'
+        justifyContent='space-between'
+        px={10}
```

Comment:
```suggestion
        px={{ base: 3, md: 4 }}
```

In order to respect mockups